src/HPND.xml

```diff
@@ -37,9 +37,9 @@
       </p>
       <optional>
         <p>
-            <alt match=".*" name="copyrightHolder2">&lt;copyright holder&gt;</alt> DISCLAIMS ALL WARRANTIES WITH REGARD TO THIS SOFTWARE, INCLUDING
+            <alt match=".*" name="copyrightHolder2">&lt;copyright holder&gt;</alt> <alt match="DISCLAIMS?" name="disclaim">DISCLAIMS</alt> ALL WARRANTIES WITH REGARD TO THIS SOFTWARE, INCLUDING
            ALL IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS<alt match="[.,]*" name="period">.</alt> IN NO EVENT SHALL
-           <alt match=".*" name="copyrightHolder3">&lt;copyright holder&gt;</alt> BE LIABLE FOR ANY SPECIAL, INDIRECT OR CONSEQUENTIAL DAMAGES
+           <alt match=".*" name="copyrightHolder3">&lt;copyright holder&gt;</alt> BE LIABLE FOR ANY SPECIAL, INDIRECT OR <alt match="CONSEQUENTIAL|COSEQUENTIAL" name"consequential">CONSEQUENTIAL</alt> DAMAGES
```

Comment:
here you are missing "=".
It should be `name="consequential"`

Reply:
sigh, I have good attention to detail, really!  Thanks (again) @xsuchy !